packages/bigquery/sodasql/dialects/bigquery_dialect.py

```diff
@@ -61,6 +62,33 @@ def create_connection(self):
         except Exception as e:
             self.try_to_raise_soda_sql_exception(e)
 
+    def __query_table(self, table_name):
+        query = f"""
+            SELECT *
+            FROM {table_name}
+            LIMIT 1
+        """
+        return query
+
+    def sql_test_connection(self, dataset_id) -> Union[Exception, bool]:
+        logging.info(f'Listing tables to check connection')
+        try:
+            tables = self.client.list_tables(dataset_id)
+            if tables:
+                print("Tables contained in '{}':".format(dataset_id))
```

Comment:
Please use logging, instead  of `print` (also in other places in the file)

Reply:
Thanks. This was solved in another commit after the PR creation